force_wfmanager/left_side_pane/mco_model_view.py

```diff
@@ -14,20 +15,18 @@ class MCOModelView(ModelView):
     #: Label to be used in the TreeEditor
     label = Str()
 
+    #: List of MCO parameters to be displayed in the TreeEditor
+    mco_parameters_representation = List(Instance(MCOParameterModelView))
+
     #: Defines if the MCO is valid or not
     valid = Bool(True)
 
-    #: List of MCO parameters to be displayed in the TreeEditor, it is a
-    #: Property so that the list in the editor is synchronized with the actual
-    #: list of parameters in the MCO model
-    mco_parameters_representation = Property(
-        List(Instance(MCOParameterModelView)),
-        depends_on="model.parameters"
-    )
-
-    def _get_mco_parameters_representation(self):
-        return [MCOParameterModelView(model=parameter)
-                for parameter in self.model.parameters]
+    @on_trait_change('model.parameters[]')
```

Comment:
Because with a property it updates lazily. But I don't want to update the list of available parameters lazily, which relies on the change of this property.